featurizers/fingerprints.py

```diff
@@ -318,16 +320,17 @@ def __init__(
         pool_multiplier: float = POOL_MULTIPLIER_DEF,
         rmsd_cutoff: float = RMSD_CUTOFF_DEF,
         max_energy_diff: float = MAX_ENERGY_DIFF_DEF,
-        force_field: float = FORCEFIELD_DEF,
+        forcefield: float = FORCEFIELD_DEF,
         get_values: bool = True,
         is_folded: bool = False,
         fold_bits: int = 1024,
         standardise: bool = True,
-        random_state: int = 0,
+        seed: int = 0,
```

Comment:
We should always use `random_state`, like Scikit-learn

Reply:
done. also moved to base class